### DISCUSSION TITLE
失败重试为什么没有起效果

### DISCUSSION THREAD

**Discussion by 15206125206** [apache/dolphinscheduler#15422] [General]
![图片1](https://github.com/apache/dolphinscheduler/assets/35686837/666f3be0-67b1-45cf-a814-b830c71615de)
![图片2](https://github.com/apache/dolphinscheduler/assets/35686837/58c4bb30-3787-4d9b-993f-a39d5d873f2b)

**Comment by Ares-W** [comment]
你用的是3.1.0版本吗？这个问题解决了没有？
